api/src/pcapi/core/finance/api.py

```diff
@@ -1779,19 +1780,19 @@ def _filter_invoiceable_cashflows(query: BaseQuery) -> BaseQuery:
     )
 
 
-def _get_cashflows_by_reimbursement_points(batch: models.CashflowBatch) -> list:
-    rows = (
-        _filter_invoiceable_cashflows(
-            db.session.query(
-                models.Cashflow.reimbursementPointId.label("reimbursement_point_id"),
-                sqla_func.array_agg(models.Cashflow.id).label("cashflow_ids"),
-            )
+def _get_cashflows_by_reimbursement_points(batch: models.CashflowBatch, only_debit_notes: bool = False) -> list:
+    query = _filter_invoiceable_cashflows(
+        db.session.query(
+            models.Cashflow.reimbursementPointId.label("reimbursement_point_id"),
+            sqla_func.array_agg(models.Cashflow.id).label("cashflow_ids"),
         )
-        .filter(models.Cashflow.batchId == batch.id)
-        .group_by(models.Cashflow.reimbursementPointId)
-    ).all()
+    ).filter(models.Cashflow.batchId == batch.id)
 
-    if not rows:
+    query = query.filter(models.Cashflow.amount > 0) if only_debit_notes else query.filter(models.Cashflow.amount < 0)
+
+    rows = (query.group_by(models.Cashflow.reimbursementPointId)).all()
```

Comment:
[détail] Les parenthèses m'ont l'air superflues : 

```suggestion
    rows = query.group_by(models.Cashflow.reimbursementPointId).all()
```